.docs/user-guide/installation.md

```diff
@@ -40,6 +40,20 @@ specify `stable`, see the additional version names below that are also valid.
 curl -sSL https://dl.bintray.com/emccode/rexray/install | sh -s -- stable
 ```
 
+#### Install a specific version
+
+You can forcibly install a particular version like this:
```

Comment:
Hi @cantbewong,

Please avoid the use of words like _you_, _I_, _we_, etc. In general the documentation should not be conversational and avoid the use of second person. This [guide](http://www.technicalauthoring.com/wiki/index.php/Using_the_correct_language#Descriptive_writing_should_be_in_the_third-person.2C_active_voice.2C_indicative) may prove useful.

An alternative approach is:

> A specific version of REX-Ray may be installed with the following command:

Please note that the word  _forcibly_ was intentionally removed. It's misleading as the installation follows the same rules as any other type, and thus is not more forceful than a standard install.